2022-02-05 15:28:39,410 INFO    MainThread:26310 [wandb_setup.py:_flush():71] setting env: {}
2022-02-05 15:28:39,410 INFO    MainThread:26310 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-05 15:28:39,410 INFO    MainThread:26310 [wandb_init.py:_log_setup():371] Logging user logs to /home/sohui/code/DTC/code/wandb/run-20220205_152839-26sfyo6q/logs/debug.log
2022-02-05 15:28:39,410 INFO    MainThread:26310 [wandb_init.py:_log_setup():372] Logging internal logs to /home/sohui/code/DTC/code/wandb/run-20220205_152839-26sfyo6q/logs/debug-internal.log
2022-02-05 15:28:39,411 INFO    MainThread:26310 [wandb_init.py:init():404] calling init triggers
2022-02-05 15:28:39,411 INFO    MainThread:26310 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'mdoel': 'DTC', 'server': 57, 'image size': '112,112,80'}
2022-02-05 15:28:39,412 INFO    MainThread:26310 [wandb_init.py:init():449] starting backend
2022-02-05 15:28:39,412 INFO    MainThread:26310 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-05 15:28:39,428 INFO    MainThread:26310 [backend.py:ensure_launched():199] starting backend process...
2022-02-05 15:28:39,437 INFO    MainThread:26310 [backend.py:ensure_launched():204] started backend process with pid: 26390
2022-02-05 15:28:39,439 INFO    MainThread:26310 [wandb_init.py:init():458] backend started and connected
2022-02-05 15:28:39,442 INFO    MainThread:26310 [wandb_init.py:init():519] updated telemetry
2022-02-05 15:28:39,443 INFO    MainThread:26310 [wandb_init.py:init():549] communicating current version
2022-02-05 15:28:40,308 INFO    MainThread:26310 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-05 15:28:40,309 INFO    MainThread:26310 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-02-05 15:28:40,559 INFO    MainThread:26310 [wandb_init.py:init():592] starting run threads in backend
2022-02-05 15:28:40,790 INFO    MainThread:26310 [wandb_run.py:_console_start():1816] atexit reg
2022-02-05 15:28:40,791 INFO    MainThread:26310 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-02-05 15:28:40,792 INFO    MainThread:26310 [wandb_run.py:_redirect():1695] Redirecting console.
2022-02-05 15:28:40,798 INFO    MainThread:26310 [wandb_run.py:_redirect():1751] Redirects installed.
2022-02-05 15:28:40,798 INFO    MainThread:26310 [wandb_init.py:init():619] run started, returning control to user process
2022-02-05 15:28:45,245 INFO    MainThread:26310 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-02-05 15:28:45,246 INFO    MainThread:26310 [wandb_run.py:_restore():1758] restore
2022-02-05 15:28:47,846 INFO    MainThread:26310 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 576
}

2022-02-05 15:28:48,108 INFO    MainThread:26310 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 576
}

2022-02-05 15:28:48,362 INFO    MainThread:26310 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 576
}

2022-02-05 15:28:48,597 INFO    MainThread:26310 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 6000
}

2022-02-05 15:28:48,701 INFO    MainThread:26310 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 6026
}

2022-02-05 15:28:48,806 INFO    MainThread:26310 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 6026
}

2022-02-05 15:28:48,911 INFO    MainThread:26310 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6026
  total_bytes: 6026
}

2022-02-05 15:28:49,015 INFO    MainThread:26310 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6026
  total_bytes: 6026
}

2022-02-05 15:28:49,119 INFO    MainThread:26310 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6026
  total_bytes: 6026
}

2022-02-05 15:28:49,223 INFO    MainThread:26310 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6026
  total_bytes: 6026
}

2022-02-05 15:28:49,327 INFO    MainThread:26310 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6026
  total_bytes: 6026
}

2022-02-05 15:28:49,604 INFO    MainThread:26310 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6026
  total_bytes: 6026
}

2022-02-05 15:28:50,133 INFO    MainThread:26310 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6026
  total_bytes: 6026
}
local_info {
}

2022-02-05 15:28:51,315 INFO    MainThread:26310 [wandb_run.py:_append_files():2184] logging synced files
